docs/degradation_and_soiling_example_pvdaq_4.ipynb

```diff
@@ -616,37 +616,6 @@
    "cell_type": "code",
    "execution_count": 17,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "0    2010-02-25 00:00:00-07:00\n",
-       "1    2010-03-08 00:00:00-07:00\n",
-       "2    2010-03-12 00:00:00-07:00\n",
-       "3    2010-04-09 00:00:00-07:00\n",
-       "4    2010-04-12 00:00:00-07:00\n",
-       "                ...           \n",
-       "59   2016-04-10 00:00:00-07:00\n",
-       "60   2016-04-16 00:00:00-07:00\n",
-       "61   2016-05-04 00:00:00-07:00\n",
-       "62   2016-07-13 00:00:00-07:00\n",
-       "63   2016-12-06 00:00:00-07:00\n",
-       "Name: start, Length: 64, dtype: datetime64[ns, Etc/GMT+7]"
-      ]
-     },
-     "execution_count": 17,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "soiling_summary['start']"
```

Comment:
Was it a conscious decision to remove this cell?  OK with me either way, just wanted to make sure it wasn't an accident.

Reply:
Confirmed. I think that was left in there from some prior debugging.